tests/acceptance/commercial/test_delta_update_module.py

```diff
@@ -174,6 +174,9 @@ def test_perform_update(
         """Perform a delta update.
 
         """
+        # TODO: Enable after next mender-binary-delta release
```

Comment:
Will you make a separate ticket for this?

Reply:
Done.